pgrx-macros/src/rewriter.rs

```diff
@@ -49,10 +49,12 @@ pub fn item_fn_without_rewrite(mut func: ItemFn) -> syn::Result<proc_macro2::Tok
         panic!("#[pg_guard] for function with generic parameters must not be combined with #[no_mangle]");
     }
 
+    attrs.push(syn::parse_quote!(#[automatically_derived]));
+    attrs.push(syn::parse_quote!(#[allow(clippy::used_underscore_binding)]));
```

Comment:
I accepted the other PR that added an `allow(clippy::lint)` because I am working on a branch that changes the code in a way that makes it non-applicable. A similar logic applies here: I would prefer to actually solve this. But I don't know if my upcoming branch can solve it.

Reply:
thx!  I think the easiest is to run `cargo clippy --workspace --bins --tests --lib --benches --examples -- -D warnings -W clippy::pedantic` on the generated code, or alternatively, add this section to the `Cargo.toml`:

```toml
[lints.rust]
unused_qualifications = "warn"

[lints.clippy]
pedantic = { level = "warn", priority = -1 }
// list any other clippy lints you want to allow, e.g. `missing_errors_doc = "allow"`
```

Reply:
I will not.

Reply:
?

Reply:
I have pushed a commit to #1731 which will silence this warning for all pgrx functions when that PR lands, because it alters the codegen strategy significantly (and so a PR fixing this would conflict with it).